nxdl.xsd

```diff
@@ -64,8 +64,8 @@
 			<xs:documentation>
 				Used for allowed names of elements and attributes.  
 				Need to be restricted to valid program variable names.  
-				Note:  This means no "-" or "." characters can be allowed and
-				you cannot start with a number.
+				Note: No ``-`` characters (among others) are allowed
+				and you cannot start or end with a period (``.``).
 				HDF4 had a 64 character limit on names 
 				(possibly including NULL) and NeXus enforces this 
 				via the ``NX_MAXNAMELEN`` variable with 
```

Comment:
Is that constant leftover from NAPI?

Reply:
yes

```
(bluesky_2020_9) prjemian@poof ~/.../NeXus/definitions $ git grep NX_MAXNAMELEN
manual/source/napi-f90.rst:``NX_MAXNAMELEN``  Maximum length of NeXus name            ``64``
nxdl.xsd:                               via the ``NX_MAXNAMELEN`` variable with 
nxdl.xsd:                       <xs:maxLength value="63" />   <!-- enforce via NX_MAXNAMELEN -->
```

Reply:
Are you suggesting we remove the statement?

Reply:
No, just that variable/constant does not exist. Various validators will enforce the limit.

Reply:
changing text to indicate it is enforced by the NAPI